--- origsrc/normalize-0.7.7/xmms-rva/Makefile.am	2005-09-09 15:19:54.000000000 -0500
+++ src/normalize-0.7.7/xmms-rva/Makefile.am	2007-07-20 16:17:56.203125000 -0500
@@ -21,7 +21,7 @@
 
 libdir = @XMMS_EFFECT_PLUGIN_DIR@
 
-librva_la_LDFLAGS = -module -avoid-version
+librva_la_LDFLAGS = -module -avoid-version -no-undefined
 #librva_la_LIBADD = @XMMS_LIBS@ @GTK_LIBS@ @LIBINTL@
 librva_la_LIBADD = @XMMS_LIBS@ @GTK_LIBS@
 
